### DISCUSSION TITLE
Auto Formatting

### DISCUSSION THREAD

**Discussion by PAPISOP** [BedrockTweaks/Files#158] [Ideas]
Is it possible to add a system to the pull request that whenever a contributor makes a pull request, then their code will be automatically formatted after the pull request has been approved to be merge

**Comment by PAPISOP** [comment]
The auto formatter will automatically run before the pull request is to be merged to the repo

**Comment by NSPC911** [comment]
can defo make a workflow but it is up to drav again
I use this in becomtweaks, but it doesn't take into account the custom indent drav wants
```cmd
npx prettier --write "**/*.{js,ts,css,json,md}"
```

**Comment by SquatchHunter** [comment]
I mean its not really a custom indent, its the stock standard for almost all IDE's out the box unless you change it so its more convention and "less work" to use tab size 4 over anything else
tbh minecraft is the one breaking convention here by using spaces and 2 instead of 4... but I digress
